src/gc-stock.c

```diff
@@ -4013,6 +4016,36 @@ JL_DLLEXPORT void jl_gc_schedule_foreign_sweepfunc(jl_ptls_t ptls, jl_value_t *o
     arraylist_push(&ptls->gc_tls.sweep_objs, obj);
 }
 
+void jl_gc_force_mark_old(jl_ptls_t ptls, jl_value_t *v) JL_NOTSAFEPOINT
```

Comment:
What is this being used for?

Reply:
Code for the older version of this that I forgot to remove